packages/measure/README.md

```diff
@@ -0,0 +1,28 @@
+# Install
```

Comment:
Add a section explaining what this packages does / what it is for?

Reply:
Done.